toml.md

```diff
@@ -293,9 +298,15 @@ For convenience, some popular characters have a compact escape sequence.
 ```
 
 Any Unicode character may be escaped with the `\xHH`, `\uHHHH`, or `\UHHHHHHHH`
-forms. The escape codes must be valid Unicode
+forms. The escape codes must be Unicode
 [scalar values](https://unicode.org/glossary/#unicode_scalar_value).
 
+Keep in mind that all TOML strings are sequences of Unicode characters, _not_
+byte sequences. For binary data, avoid using these escape codes. Instead,
+external binary-to-text encoding strategies, like hexadecimal sequences or
```

Comment:
I'm not quite convinced of the new mention of "hexadecimal sequences". If one wants to mention an alternative to Base64, why not mention [Base85](https://en.wikipedia.org/wiki/Ascii85#RFC_1924_version), which is more rather than less compact than Base64, is used by Git and supported by the Python standard library, and even has its own RFC (if originally an April Fools RFC). Or just stick to Base64, as before.

Otherwise this looks really good now!